chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/summarystatistics/data/StatisticsSnapshot.java

```diff
@@ -0,0 +1,108 @@
+package gov.healthit.chpl.scheduler.job.summarystatistics.data;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class StatisticsSnapshot implements Serializable {
+    private static final long serialVersionUID = 6977674702447509179L;
+
+    private List<CertificationBodyStatusStatistic> developerCountsByStatus;
+    private List<CertificationBodyStatusStatistic> productCountsByStatus;
+    private List<CertificationBodyStatusStatistic> listingCountsByStatus;
+
+    //////////////////////////////////////////////////////////////////////
+
+    private Long surveillanceAllStatusTotal;
+    private Statistic surveillanceOpenStatus;
+    private Long surveillanceClosedStatusTotal;
+    private Long surveillanceAvgTimeToClose;
+
+    //////////////////////////////////////////////////////////////////////
+
+    private Long nonconfStatusAllTotal;
```

Comment:
I think we spell "non-conformity" with a dash, which in camel casing would mean that "conf" should be capitalized

```suggestion
    private Long nonConfStatusAllTotal;
```